docs/infra/website.md

```diff
@@ -0,0 +1,196 @@
+
+This section describes how to add new content to the FOSS@MAGIC website.
+
+Add a new `.md` file to one of the `_posts/` folders for a category.
+Store assets (a.k.a. pictures, PDFs, or other downloadable files) in a folder within that category (e.g. `projects/assets/`).
+Include the date in asset file names so it is clear what references it.
+
+Content categories include the following:
+
+* [Announcements](https://github.com/FOSSRIT/fossrit.github.io/tree/master/announcements)
+* [Events](https://github.com/FOSSRIT/fossrit.github.io/tree/master/events)
+* [Projects](https://github.com/FOSSRIT/fossrit.github.io/tree/master/projects)
+
+### Git work flow
+
+1. Fork the repository to your own Github account (go to [FOSSRIT/fossrit.github.io](https://github.com/FOSSRIT/fossrit.github.io) and click "fork")
+2. Clone repository locally to your workstation
+3. Create/add files
+4. Git commit
+5. Git push to your fork
+6. Open a pull request against `master` branch of FOSSRIT/fossrit.github.io repository
+7. When pull request is merged, site updates in a couple of minutes
+
+### Jekyll and Front Matter
+
+Jekyll is what builds the site.
+It renders Markdown and handles templating.
+_Jekyll requires that all post pages start with a valid date in the syntax `YYYY-MM-DD-<title>.md`_.
+
+Jekyll uses something called [Front Matter](https://jekyllrb.com/docs/front-matter/) to know more information about the page.
+Front Matter defines what the page looks like and metadata such as the title, author, and event details (where applicable).
+
+Frontmatter should be at the top of a post and should be surrounded by triple-dashes (`---`).
+_All times are of the timezone `America_New_York`._
+
+Additionally, Jekyll uses the string `<!--more-->` to define where the summary of pages end.
+This is used when showing post previews on the site.
+Optionally, the `excerpt` frontmatter can be defined instead to override this (if using an HTML page instead of Markdown in special cases, the `excerpt` frontmatter _should be used_ instead of `<!--more-->`).
+Please use this functionality to keep lists of posts readable.
+
+#### Example post
+
+```yaml
+---
+title: Example thing
+layout: post
+---
+Hello FOSS folks!
+
+This week...
+
+<!--more-->
+
+Other things happening.
+
+- Signature
+```
+
+### How to create announcements
+
+Announcements spread the word about what is happening in the community.
+Posts in this category are added to an RSS Feed (at [fossrit.github.io/feeds/latest.xml](https://fossrit.github.io/feeds/latest.xml)).
+
+* Create a new Markdown file in `announcements/_posts/` named with the convention `YYYY-MM-DD-<title>.md`
+
+#### Example announcement
+
+See [2019-12-17-spring-2020-volunteers.md](https://github.com/FOSSRIT/fossrit.github.io/blob/67282a55a6330a5c61397eaeac92ef9c025f0432/announcements/_posts/2019-12-17-spring-2020-volunteers.md):
+
+```yaml
+---
+author: Justin W. Flory
+title: "Call for Volunteers: Spring 2020"
+layout: post
+date: 2019-12-17
```

Comment:
`date` is not needed since Jekyll reads it from the name of the file. I would recommend leaving this out since while Jekyll can take a time argument here, the site generally assumes this to be date-only and defining it again is redundant. Jekyll's [posts](https://jekyllrb.com/docs/posts/) doc doesn't include this, and the front matter docs advocate "don't repeat yourself" with front-matter.

Reply:
Addressed in commit b4a213117093c27d880b65cd6835d0bd46d14663.